wazo_confd/plugins/moh/schema.py

```diff
@@ -1,16 +1,18 @@
-# Copyright 2017-2019 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2017-2021 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from marshmallow import fields
-from marshmallow.validate import Length, OneOf
+import logging
 
-from wazo_confd.helpers.mallow import BaseSchema, Link, ListLink, AsteriskSection
+from marshmallow import fields, pre_load
+from marshmallow.validate import Length, OneOf, Regexp
 
-# the regex is more restrictive since the name is used both for the Asterisk
-# section and the directory on the file system
-moh_name_validator = AsteriskSection(
-    max_length=20, regex=r'^[a-zA-Z0-9][-_.a-zA-Z0-9]*$'
-)
+from wazo_confd.helpers.mallow import BaseSchema, Link, ListLink
+
+logger = logging.getLogger(__name__)
+
+# The label is going to end in musiconhold.conf and used in agi.verbose calls.
+# Try not to be too permissive with it
+LABEL_REGEX = r'^[-_.a-zA-Z0-9 ]+$'
```

Comment:
Really? Or this behavior has changed with the PR?
Because the purpose of label is indeed to be very permissive

Reply:
> Really? Or this behavior has changed with the PR? Because the purpose of label is indeed to be very permissive

That's a pretty copy-paste from groups :) I'm not sure why it has this limitation though, so I put it there also. It may not be necessary at all.

Reply:
According the confgend, is not in the musiconhold.conf ;) 
https://github.com/wazo-platform/wazo-confgend/blob/master/wazo_confgend/templates/asterisk/musiconhold.jinja

But well ... groups are wrong to restrict this value !!